Composer/packages/client/src/extension-container/ExtensionContainer.tsx

```diff
@@ -157,7 +169,9 @@ function ExtensionContainer() {
 
   const RealEditor = shellData.data ? getEditor() : null;
 
-  return RealEditor && <RealEditor {...shellData} onChange={shellApi.saveData} shellApi={shellApi} />;
+  return (
+    RealEditor && <RealEditor key={forceUpdateCount} {...shellData} onChange={shellApi.saveData} shellApi={shellApi} />
+  );
```

Comment:
So this will completely reset all internal state in the extension right? I'm not sure we want that.

Reply:
I have removed it now.